docs/site/lib/constants.ts

```diff
@@ -1,6 +1,6 @@
 export const PRODUCT_SLOGANS = {
   turbo:
-    "Turbo is an incremental bundler and build system optimized for JavaScript and TypeScript, written in Rust.",
+    "Turborepo is a build system optimized for JavaScript and TypeScript, written in Rust.",
   repo: "The build system that makes ship happen.",
```

Comment:
Do we still use both of these slogans in different places?

Reply:
I see only one other reference to "bundler" and its a keeper until I redesign the home page.

Reply:
Oh, or did you mean more generally? We use both of these here and there but do need to audit/rework/pick one in time, yes.